Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Dockerfile to build and test DeepDive inside a container
+#
+# `make build-in-container test-in-container` uses master image built by this.
+# `util/build/docker/` contains utilities relevant to this.
+FROM ubuntu
```

Comment:
I don't know how "ubuntu" is resolved -- maybe it's resolved to "latest" so in October it'll be ubuntu:16.10 right after the release. But if during the next 18 months, production envs stick to 16.04 (which is the most likely scenario), then Travis would be testing against an env different from prod during that time. When DD breaks on 16.04 but not on 16.10, Travis wouldn't catch that.